pubspec.yaml

```diff
@@ -1,34 +1,30 @@
 name: codemod
-version: 0.2.0
+version: 0.3.0
```

Comment:
Any reason to not go to 1.0.0? 🙂

Reply:
I was going to follow this up with a null-safe version, so I was sort of waiting for that

Reply:
Ooh good point; that's fair!